incubator/kubernetes-vault/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
If you change this, you'll also need to add a `spec.selector`. Make it `apps/v1beta2`.